libs/core/src/lib/carousel/carousel.component.html

```diff
@@ -40,7 +40,10 @@
             <ng-container *ngTemplateOutlet="buttonLeft"></ng-container>
         </ng-container>
 
-        <ng-container *ngTemplateOutlet="pageIndicators"></ng-container>
+        <ng-container
+            *ngIf="_showNavigationButtonInPageIndicatorContainer"
+            *ngTemplateOutlet="pageIndicators"
```

Comment:
```suggestion
            [ngTemplateOutlet]="pageIndicators"
```

because https://app.netlify.com/sites/fundamental-ngx/deploys/621e32987d93f700078105d1#L570

Reply:
Done